styles.css

```diff
@@ -0,0 +1,669 @@
+  flex-direction: column;
+  align-items: flex-start;
+  margin-left: 20px;
+  margin-bottom: 0;
+}
+
+.PrimaryText {
+  width: 327px;
+  height: 124px;
+  margin-bottom: 10px;
+}
+
+.PrimaryText h1 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  color: #ff6b00;
+  letter-spacing: 0.37px;
+  justify-content: center;
+  margin-bottom: 10px;
+}
+
+.SupportingText {
+  width: 327px;
+  height: 27px;
+  margin: 20px 0;
+}
+
+.SupportingText h2 {
+  justify-content: center;
+  align-items: center;
+  width: 327px;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  color: #fff;
+}
+
+.discription {
+  width: 327px;
+  height: 123px;
+  margin-bottom: 20px;
+}
+
+.discription p {
+  width: 327px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.SocialMedia {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  text-decoration: none;
+  gap: 20px;
+  width: 200px;
+  list-style: none;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+ul .SocialMedia {
+  width: 200px;
+  height: 40px;
+}
+
+.SocialMedia img {
+  height: 24px;
+  width: 24px;
+  border-radius: 0;
+}
+
+.works {
+  width: 375px;
+  background-color: #362315;
+}
+
+.RecentWorks {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  height: 107px;
+  padding-top: 180px;
+  justify-content: center;
+  align-items: center;
+}
+
+.RecentWorks h2 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 450;
+  font-size: 38px;
+  line-height: 52px;
+  margin-left: 24px;
+}
+
+#line {
+  width: 327px;
+  padding: 0 24px;
+  border: 1px solid #212121;
+  align-items: center;
+  justify-content: center;
+  margin-left: 24px;
+}
+
+.MultiPostStories {
+  display: flex;
+  flex-direction: column;
+  margin-top: 160px;
+  justify-content: center;
+  align-items: center;
+  width: 375px;
+
+  /* background-color: #ff6b00; */
+}
+
+.MultiPostStories img {
+  width: 327px;
+  height: 237px;
+  justify-content: center;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.MultiPostStories h3 {
+  width: 300px;
+  height: 26px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+
+  /* font-weight: 400; */
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  justify-content: center;
+  color: #091e42;
+  margin-left: 15px;
+}
+
+.MultiPostStories p {
+  padding: 0 24px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  margin: 0;
+}
+
+.languageBoxes {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 10px;
+  padding-left: 0;
+  margin: 5px;
+  margin-bottom: 0;
+  margin-top: 0;
+  list-style: none;
+  margin-left: 15px;
+}
+
```

Comment:
- [ ] Great job so far on this project, but it could be better if you put some space between your paragraph and your boxes-languages for your project to look awesome ,kindly organize this as the Figma template shows:wink: 
Your design :point_down: 
![paraggraph](https://user-images.githubusercontent.com/88894525/198014628-257dcd56-c22c-47d0-b159-437541345dd2.png)

Figma Design :point_down: 
![figmaparagraph](https://user-images.githubusercontent.com/88894525/198014737-b74f19e1-99ea-4b10-9847-7afa8957e2b4.png)

Reply:
- [ ] you have just stopped the project for nothing, this is easy to understand that I have used the uppercase instead of the lower case which is very minor 
- [ ] the spacing of word is the one provided and if I use more than that it will over lop.
- [ ] all in all there is no strong or week reason for stopping the project like most of your say are optional